webapp/src/views/projects/translations/TranslationOpened.tsx

```diff
@@ -134,17 +136,17 @@ export const TranslationOpened: React.FC<Props> = ({
     }
   };
 
-  const baseLanguage = useTranslationsSelector((v) =>
+  const sourceLanguage = useTranslationsSelector((v) =>
     v.languages?.find((l) => l.base)
   )?.tag;
-  const baseTranslation =
-    baseLanguage && keyData.translations[baseLanguage]?.text;
+  const sourceText =
+    sourceLanguage && keyData.translations[sourceLanguage]?.text;
 
   const data = useTranslationTools({
     projectId: project.id,
     keyId: keyData.keyId,
     targetLanguageId: language.id,
-    baseText: baseTranslation,
+    baseText: sourceText,
```

Comment:
It should be sourceText all over the codebase, but I won't refactor all unrelated code in this PR.

Reply:
The term `baseText` is contained 27 times in the codebase, baseTranslation 85 times and baseLangauge 162 times. In the code it can be refactored, right. But these terms are also exposed in public API endpoints. Renaming the some of the properties would lead to breaking changes in the API. 

That being said, we are currently unable to fix this terminology issue. And we shouldn't make the code inconsistent. So please change it to make it consistent.